github-actions/org-file-sync/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Organization File Sync'
+description: 'Syncronize files throughout the Github Organization'
+author: 'Angular'
+inputs:
+  angular-robot-key:
```

Comment:
One important question I think: We now use Renovate with the `repo` and `workflow` access token for `angular-robot`. With the robot account being used in such cases (with write access to default branches), we are exposing more permissions to the Renovate processes. Should we have a separate Renovate account instead?